### PR TITLE
[MRXN23-237] (feat): scenario list 

### PR DESCRIPTION
**Pull request by anamontiaga**
### Overview

New layout for project panel scenario list

### Designs
[Scenario list view prototype with no scenarios](https://www.figma.com/proto/0xD4gR9pCqU7NROnlSW6ck/Marxan-Layout---Redesign?node-id=568-40437&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=568%3A40417&show-proto-sidebar=1)
[Scenario list view prototype with some scenarios](https://www.figma.com/proto/0xD4gR9pCqU7NROnlSW6ck/Marxan-Layout---Redesign?node-id=521-57939&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=521%3A58646)

### Testing instructions
- Create a project and check you can edit name and description, publish and unpublish it and download. Also try to add someone to this project.
- Create a project and check if you are able to create one scenario or navigate to upload features, protected areas or cost surface tabs.
- Once you have some scenarios, check if you can see the settings, duplicate and download each scenario and also see its detail. 

### Feature relevant tickets

[MRXN23-237](https://vizzuality.atlassian.net/browse/MRXN23-237)

[MRXN23-237]: https://vizzuality.atlassian.net/browse/MRXN23-237?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/layout/project/navigation/index.tsx [hunk 1]
```diff
@@ -101,6 +101,13 @@ export const Navigation = (): JSX.Element => {
     });
   }, []);
 
+  useEffect(() => {
```
Comment (anamontiaga) [inline app/layout/project/navigation/index.tsx:104]: Maybe we should save menu values in store

File: app/layout/project/sidebar/project/header/contributors/index.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  const { data: projectUsers } = useProjectUsers(pid);
+
+  const projectUsersVisibleSize = 3;
+  const projectUsersVisible = projectUsers?.slice(0, projectUsersVisibleSize);
+
+  const handleClick = useCallback(() => {
+    setOpen(!open);
+  }, [open, setOpen]);
+
+  const handleClickOutside = useCallback(
```
Comment (andresgnlez) [inline app/layout/project/sidebar/project/header/contributors/index.tsx:37]: Check if we can replace this behaviour with the Popover component.

File: app/types/scenario.ts [hunk 3]
```diff
@@ -1,4 +1,19 @@
 export interface Scenario {
   id: string;
-  // todo: keep typing scenario as much as posible
+  name: string;
+  warnings: boolean;
+  progress?: number;
+  lastUpdate: string;
+  jobs?: Record<string, any>[];
```
Comment (andresgnlez) [inline app/types/scenario.ts:7]: Check Job type that we already have

File: app/types/scenario.ts [hunk 4]
```diff
@@ -1,4 +1,19 @@
...
+  runStatus: 'created' | 'running' | 'done' | 'failure';
+  lock?: Record<string, any>;
+  lastUpdateDistance: string;
+  className?: string;
```
Comment (andresgnlez) [inline app/types/scenario.ts:11]: This shouldn't be here.

**Pull request merged**